internal/template-bundle/bundle.go

```diff
@@ -47,6 +54,26 @@ func ReadBundle(filename string) (Bundle, error) {
 	}, nil
 }
 
+func RetrieveCommonTemplatesBundleFile(templateBundleDir string) (string, error) {
+	archDependentFileName := filepath.Join(templateBundleDir, commonTemplatesFilePrefix+runtime.GOARCH+"-"+common_templates.Version+commonTemplatesFileSuffix)
+	archIndependentFileName := filepath.Join(templateBundleDir, commonTemplatesFilePrefix+common_templates.Version+commonTemplatesFileSuffix)
+	commonTemplatesExistingFiles := filterExistingTemplateFiles([]string{archDependentFileName, archIndependentFileName})
```

Comment:
Can you simplify this by not using a filter function? IMHO directly checking if the 2 files exists is more readable.
```
if _, err := os.Stat(archDependentFileName); err == nil {
    return archDependentFileName, nil
}

if _, err := os.Stat(archIndependentFileName); err == nil {
    return archIndependentFileName, nil
}
```

Reply:
Done